rust/ql/src/queries/diagnostics/SsaConsistencyCounts.ql

```diff
@@ -0,0 +1,14 @@
+/**
+ * @name Static single assignment inconsistency counts
+ * @description Counts the number of static single assignment inconsistencies of each type.  This query is intended for internal use.
+ * @kind diagnostic
+ * @id rust/diagnostics/ssa-consistency-counts
+ */
+
+private import codeql.rust.dataflow.internal.SsaImpl as SsaImpl
```

Comment:
Perhaps just `private import codeql.rust.dataflow.internal.SsaImpl::Consistency`?

Reply:
That exact import causes all the `query predicates` in `SsaImpl::Consistency` to apply, breaking this query - but `private import codeql.rust.dataflow.internal.SsaImpl::Consistency as SsaConsistency` works and is more precise. 
 I've updated the query.